### DISCUSSION TITLE
No Auth options?

### DISCUSSION THREAD

**Discussion by Fran-Rg** [plausible/analytics#4364] [Self-Hosted Support]
As explained on https://github.com/plausible/analytics/discussions/383 + https://github.com/plausible/analytics/pull/3170 + https://github.com/plausible/analytics/pull/3706 the support for OIDC and other alternative auth methods is far from becoming a thing yet.

Thus I'm interested in setting the domain as no auth and add a blanket SSO layer on top except for the paths that need to be available for all. Is there a way to do that?

**Comment by ruslandoga** [comment]
👋 @Fran-Rg 

I don't think that option exists right now.
